MIGRATION_GUIDE.md

```diff
@@ -30,6 +30,28 @@ Please adjust your Terraform configuration files.
 ### *(behavior change)* Provider configuration rework
 On our road to v1, we have decided to rework configuration to address the most common issues (see a [roadmap entry](https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/ROADMAP.md#providers-configuration-rework)). We have created a list of topics we wanted to address before v1. We will prepare an announcement soon. The following subsections describe the things addressed in the v0.98.0.
 
+#### *(behavior change)* new fields
+We have added new fields to match the ones in [the driver](https://pkg.go.dev/github.com/snowflakedb/gosnowflake#Config) and to simplify setting account name. Specifically:
+- `include_retry_reason`, `max_retry_count`, `driver_tracing`, `tmp_directory_path` and `disable_console_login` are the new fields that are supported in the driver
+- `disable_saml_url_check` will be added to the provider after upgrading the driver
+- `account_name` and `organization_name` were added to improve handling account names. Read more in [docs](https://docs.snowflake.com/en/user-guide/admin-account-identifier#using-an-account-name-as-an-identifier).
+
+#### *(behavior change)* deprecated fields
+Because of new fields `account_name` and `organization_name`, `account` is now deprecated. Please adjust your configurations from
```

Comment:
Done.

Reply:
nit (for next PR): `It will be removed before v1.` -> `It will be removed with the v1 release.` (before V1 is inaccurate and AFAIK we do not plan to remove it before the V1)